gen/generate.go

```diff
@@ -226,12 +256,17 @@ func generateModule(
 	builder *generateServiceBuilder,
 	o *Options,
 ) (outputFilepath string, contents []byte, err error) {
+	if err := duplicateAfterNormalization(m.ThriftPath); err != nil {
```

Comment:
This is normalizing twice. The logic is effectively,

    newPath = normalize(path)
    if checkStuff(newPath) {
    }
    
    newPath = normalize(path)

I would drop `duplicateAfterNormalization` and move the duplication check here.

Reply:
done